### PR TITLE
[zh-cn/chore] link cleanup

### PR DESCRIPTION
**Pull request by jasonren0403**
### Description

Currently a lot of flaws are caused by bare links like /zh-CN/Learn/... and /zh-CN/Apps/..., and `docs` should be prefixed after language codes. Do cleanup for link not started with /zh-CN/docs for /games and /glossary

### Motivation

reduce redirects and flaws

### Additional details

used regex `\/zh-CN\/(?!docs\/).*` to find the link, the regex is not good enough to distinguish local links and full links, it can be better

### CODE REVIEW

File: files/zh-cn/glossary/mvc/index.md [hunk 1]
```diff
@@ -11,12 +11,6 @@ slug: Glossary/MVC
...
+## 参见
 
-### General knowledge
-
-- [Model–view–controller](https://zh.wikipedia.org/wiki/Model–view–controller) on Wikipedia
-
-### Learning MVC
-
-- [MVC architecture](/zh-CN/Apps/Fundamentals/Modern_web_app_architecture/MVC_architecture)
+- 维基百科上的 [Model–view–controller](https://zh.wikipedia.org/wiki/Model–view–controller) 词条
```
Comment (yin1999) [inline files/zh-cn/glossary/mvc/index.md:16]:
```suggestion
- 维基百科上的 [MVC 模式](https://zh.wikipedia.org/wiki/MVC)词条
```

File: files/zh-cn/glossary/ftp/index.md [hunk 2]
```diff
@@ -7,9 +7,7 @@ slug: Glossary/FTP
...
 **FTP** 是 TCP/IP 协议组中的协议之一。FTP 协议包括两个组成部分，其一为[FTP 服务器](https://baike.sogou.com/lemma/ShowInnerLink.htm?lemmaId=449752&ss_c=ssc.citiao.link)，其二为 FTP 客户端。其中 FTP 服务器用来存储文件，用户可以使用 FTP 客户端通过 FTP 协议访问位于 FTP 服务器上的资源。在开发网站的时候，通常利用 FTP 协议把网页或程序传到[Web 服务器](https://baike.sogou.com/lemma/ShowInnerLink.htm?lemmaId=267249&ss_c=ssc.citiao.link)上。此外，由于 FTP 传输效率非常高，在网络上传输大的文件时，一般也采用该协议。
 
-## 了解更多
+## 参见
 
-### 总体了解
-
-- [Beginner's guide to uploading files via FTP](/zh-CN/Learn/Upload_files_to_a_web_server)
+- [通过 FTP 上传文件的初学者教程](/zh-CN/docs/Learn/Common_questions/Tools_and_setup/Upload_files_to_a_web_server)
 - 维基百科上的 [FTP](https://zh.wikipedia.org/wiki/File_Transfer_Protocol)
```
Comment (yin1999) [inline files/zh-cn/glossary/ftp/index.md:13]:
```suggestion
- 维基百科上的 [FTP](https://zh.wikipedia.org/wiki/文件传输协议)
```

File: files/zh-cn/glossary/certified/index.md [hunk 3]
```diff
@@ -7,13 +7,6 @@ slug: Glossary/Certified
 
 关于{{glossary("密码学")}}中证书的细节，请参考{{glossary("数字认证证书")}}。
 
-## 了解更多
+## 参见
 
-### 一般性知识
-
-- [信息安全教程](/zh-CN/Learn/tutorial/Information_Security_Basics)
 - 维基百科中的[证书](https://zh.wikipedia.org/wiki/Professional_certification_(computer_technology)#Information_systems_security)
```
Comment (yin1999) [inline files/zh-cn/glossary/certified/index.md:12]: 貌似 `https://en.wikipedia.org/wiki/Professional_certification_(computer_technology)#Information_systems_security` 就已经是过时的了，方便去 mdn/content 修复这个链接吗

Comment (jasonren0403) [reply files/zh-cn/glossary/certified/index.md:12]: 刚关电脑😂，要不你来提？

Comment (yin1999) [reply files/zh-cn/glossary/certified/index.md:12]: mdn/content#26050

Comment (yin1999) [reply files/zh-cn/glossary/certified/index.md:12]: 中文维基貌似没有这个小节，就不作小节链接了

```suggestion
- 维基百科中的[证书](https://zh.wikipedia.org/wiki/專業認證)
```

**Review by yin1999 — APPROVED** [review]

**Pull request merged**